stable/jaeger-operator/templates/psp.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if and .Values.rbac.create .Values.rbac.pspEnabled }}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ include "jaeger-operator.fullname" . }}-operator-psp
+  labels:
+    app: {{ include "jaeger-operator.fullname" . }}-operator
```

Comment:
Add the standard labels as the others templates

Reply:
I added all labels in `_helpers.tpl` file as `jaeger-operator.labels` and included here. 
I thought it may be good to keep the labels in one place and just 'inject' them in every file. Should I get rid of the template and add them directly into the file? 
(if it can stay in `_helpers.tpl` file I can modify all other resource templates to include `jaeger-operator.labels` - let me know what do you think)

Reply:
sounds good thanks!